docs/howto/connection/index.rst

```diff
@@ -221,6 +221,13 @@ To fix this, you can encode with :py:meth:`~urllib.parse.quote_plus`:
     >>> print(c.password)
     my-pa/ssword
 
+Securing Connections
+--------------------
+
+Airflow uses `Fernet <https://github.com/fernet/spec/>`__ to encrypt passwords in the connection
+configuration. It guarantees that a password encrypted using it cannot be manipulated or read without the key.
+
+For information on configuring Fernet, look at :ref:`Generating a Connection URI <security/fernet>`.
```

Comment:
We had https://airflow.apache.org/docs/stable/howto/secure-connections.html#securing-connections . Did these get lost in the refactor?

Reply:
It has been moved to security page: 
https://airflow.readthedocs.io/en/latest/security.html#securing-connections
Now I want to add a link to the security page on the configuration page.  Many people may not notice the Fernet configuration if it is in the security page.

Reply:
I would like to divide the security page into several smaller subpages, but this is a different task.

Reply:
The similar reference we have on "[Using the CLI](https://airflow.readthedocs.io/en/latest/usage-cli.html#creating-a-connection)" page. We have many guides on this page, but guide about connections in "[Managing Connections](https://airflow.readthedocs.io/en/latest/howto/connection/index.html#connection-cli)"